txn_writer.go

```diff
@@ -21,15 +21,43 @@ import (
 // Example string to be written to the manifest file
 // if no dependencies are found in the project
 // during `dep init`
-const exampleToml = `
-# Example:
+const exampleTOML = `
+## EXAMPLES & DOCS - safe to delete!
+#
+## Dependencies define constraints on how dependent projects should be
```

Comment:
yeah that's fair, i was trying to make more of a distinction than we really needed